src/pages/plans/Plans.tsx

```diff
@@ -36,6 +34,7 @@ const Plans: FC<PlansProps> = () => {
                 {plansTabs.map(({ name, icon: Icon, id }) => {
                   return (
                     <ListItem disablePadding>
+                      key={id}
```

Comment:
Added outside of JSX
![image](https://github.com/Zuzana-bor/fit-body/assets/42697958/b111470f-7178-4a76-97f0-dd34252d0aaa)